edkrepo/commands/humble/f2f_cherry_pick_humble.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+#
+## @file
+# f2f_cherry_pick_humble.py
+#
+# Copyright (c) 2020 - 2021, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+''' Contains informational and error messages outputted by
+the run_command functions of Edkrepo commands

```

Comment:
I think this comment should be clarified to say that it is for the EdkRepo f2f command not generic commands

Reply:
Please see new patch